app/assets/javascripts/admin/addon/controllers/admin-emojis.js

```diff
@@ -44,6 +44,19 @@ export default Controller.extend({
     },
   }),
 
+  _highlightEmojiList() {
```

Comment:
Any "easy" acceptance tests we can squeeze in for this?

Reply:
I very much doubt it :sweat: